src/test/kotlin/org/ptss/support/core/context/PostgresContextTestContainer.kt

```diff
@@ -0,0 +1,36 @@
+package org.ptss.support.core.context
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager
+import org.testcontainers.containers.PostgreSQLContainer
+import org.testcontainers.utility.DockerImageName
+
+class PostgresContextTestContainer : QuarkusTestResourceLifecycleManager {
+    private lateinit var postgresContainer: PostgreSQLContainer<*>
+
+    override fun start(): Map<String, String> {
+        postgresContainer = PostgreSQLContainer(DockerImageName.parse("postgres:15-alpine"))
+            .withDatabaseName("test_database")
+            .withUsername("test")
+            .withPassword("test")
+
+        postgresContainer.start()
+
+        // Create schema after container starts
+        postgresContainer.createConnection("").use { connection ->
+            connection.createStatement().execute("CREATE SCHEMA IF NOT EXISTS tool_service")
+        }
```

Comment:
Is dit nodig? Wrm moet de schema hier gecreëerd worden? Dit zou je ORM toch al moeten handlen? 

Verder opnieuw ook dezelfde concern als mijn andere comment:
> Kan "tool_service" uit de .env of application.properties gehaald worden?

Reply:
Goed gespot, heb het veranderd. Had eerst namelijk het probleem dat het zich af vroeg waar `tool_service.tools` is. Maar blijkbaar na wat dingen veranderd te hebben hoefd dat dus niet meer.